# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/ipc/RemoteAccessibleBase.h
# Commit: 227139b26cc2
# Full Hash: 227139b26cc266da35f3e9a9da9f61e18c9d0248
# Author: Morgan Rae Reschenberg <mreschenberg@mozilla.com>
# Date: 2023-02-01 09:43:56
# Regressor Bug: 1809836
# File Overlap Count: 1
# Description:
#   Bug 1809836: Avoid adding scroll offsets when computing bounds for position:fixed accs r=Jamie
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D166730
# ==============================================================================

diff -r 8bd422f41771 -r 227139b26cc2 accessible/ipc/RemoteAccessibleBase.h
--- a/accessible/ipc/RemoteAccessibleBase.h	Tue Jan 31 23:49:06 2023 +0000
+++ b/accessible/ipc/RemoteAccessibleBase.h	Wed Feb 01 01:23:10 2023 +0000
@@ -433,6 +433,7 @@
   void ApplyScrollOffset(nsRect& aBounds) const;
   void ApplyCrossDocOffset(nsRect& aBounds) const;
   LayoutDeviceIntRect BoundsWithOffset(Maybe<nsRect> aOffset) const;
+  bool IsFixedPos() const;
 
   virtual void ARIAGroupPosition(int32_t* aLevel, int32_t* aSetSize,
                                  int32_t* aPosInSet) const override;